conda_smithy/feedstock_content/LICENSE

```diff
@@ -1,3 +1,7 @@
+A separate license may cover the recipe itself.  Please see the recipe
```

Comment:
I'm not sure what the value of this line is. It is adding confusion, and typically I'd avoid the word "may". Instead, I'd expect recipes that had been LICENSED differently to have a LICENSE in the recipe dir...